var/spack/repos/builtin/packages/cellpose/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Cellpose(PythonPackage):
+    """A generalist algorithm for cell and nucleus segmentation that can be optimized for
+    your own data"""
+
+    homepage = "https://cellpose.readthedocs.io/"
+    pypi = "cellpose/cellpose-2.2.3.tar.gz"
+
+    license("BSD-3-Clause", checked_by="A-N-Other")
+
+    version("2.2.3", sha256="7ff63cb60a154ce3c0a17ff05ed27d2aaaa1a24a4c7a8160c0a4c15443366618")
+
+    variant("cuda", default=True, description="Require py-torch built with CUDA support")
```

Comment:
I don't see a need for a variant here. Users can always request:
```console
> spack install cellpose ^py-torch+cuda
```
We generally frown upon variant forwarding because it leads to proliferation of variants and isn't actually necessary to get what you need.

Reply:
Sure thing. Dropped that.